R/dummy.R

```diff
@@ -231,13 +231,14 @@ prep.step_dummy <- function(x, training, info = NULL, ...) {
   )
 }
 
-warn_new_levels <- function(dat, lvl) {
+warn_new_levels <- function(dat, lvl, ...) {
```

Comment:
In vctrs, to make this more readable we pass a single `details` argument rather than `...`. That way if we look back at this in a month we aren't confused about why there are dots here